src/components/stimuli/inputcomponents/utils.ts

```diff
@@ -19,8 +19,16 @@ const generateValidation = (responses: Response[], id: string) => {
         ...validateObj,
         [`${id}/${response.id}`]: (value: string | string[]) => {
           if (Array.isArray(value)) {
+            if(response.requiredValue && !Array.isArray(response.requiredValue)) {
+              return 'Incorrect required value';
+            } else if (response.requiredValue && Array.isArray(response.requiredValue)) {
+              return response.requiredValue.length === value.length && response.requiredValue.every((val, index) => val === value[index]) ? null : 'Incorrect input';
```

Comment:
This requires that the arrays have the same order. Is that guaranteed?

Reply:
True, changed to sort them first (in a new array, not sure if I can safely sort the response/value array in place?)